src/position/template/contract-position.template.position-fetcher.ts

```diff
@@ -215,4 +222,55 @@ export abstract class ContractPositionTemplatePositionFetcher<
 
     return balances;
   }
+
+  async getBalancesRaw(address: string): Promise<RawContractPositionBalance[]> {
+    const multicall = this.appToolkit.getMulticall(this.network);
+    const contractPositions = await this.appToolkit.getAppContractPositions<V>({
+      appId: this.appId,
+      network: this.network,
+      groupIds: [this.groupId],
+    });
+
+    return Promise.all(
+      contractPositions.map(async contractPosition => {
+        const contract = multicall.wrap(this.getContract(contractPosition.address));
+        const balancesRaw = await this.getTokenBalancesPerPosition({ address, contract, contractPosition, multicall });
+        return {
+          groupId: this.groupId,
+          key: this.appToolkit.getPositionKey(contractPosition),
+          tokens: contractPosition.tokens.map((token, i) => ({
+            key: this.appToolkit.getPositionKey(token),
+            balance: balancesRaw[i].toString(),
+          })),
+        };
+      }),
+    );
+  }
+
+  async drillRawBalances(balances: RawContractPositionBalance[]): Promise<ContractPositionBalance<V>[]> {
```

Comment:
nit: the method name here could be symmetric. E.g. `getBalancesRaw` & `drillBalancesRaw`. Also could be the same as the ones defined in `src/position/template/app-token.template.position-fetcher.ts`.